### PR TITLE
Update default gravatar & use rounded version of it as placeholder

### LINKED ISSUE #2673: Update mystery man gravatar and change it to be round
We would like to update the default gravatar to: https://cloudup.com/cosBX-nwHaE It should also be rounded everywhere it's used.

### PR DESCRIPTION
**Pull request by oguzkocer**
Fixes #2673. Screenshot from the Me page: https://cloudup.com/cTQA3ps6Unp

Since we need the circular version of this placeholder, I've added a helper method to set the image to default gravatar which takes care of the rounding. I am not sure how to test CommentUserNoteBlock & UserNoteBlock changes, but since we always use the rounded versions of the gravatars in the notifications, I am hoping this change will be for the better.

It's a little unrelated but I've also updated the blavatar placeholder in `reader_blog_info_view`.

### CODE REVIEW

**Comment by oguzkocer** [comment]
Per our discussion on Slack, we have decided to remove the gravatar image altogether from Me page when the user is logged in with a self-hosted site(s). This is a temporary solution, and we'd like to handle that better in the next releases.

File: WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java [hunk 1]
```diff
@@ -121,11 +122,11 @@ public void onClick(View view) {
                 }
             });
         } else {
-            mAvatarImageView.setImageResource(R.drawable.gravatar_placeholder);
             removeDropShadow();
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java]: `removeDropShadow()` is no longer necessary.

File: WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public View configureView(View view) {
                 noteBlockHolder.avatarImageView.setOnTouchListener(null);
             }
         } else {
-            noteBlockHolder.avatarImageView.setImageResource(R.drawable.gravatar_placeholder);
+            noteBlockHolder.avatarImageView.showDefaultGravatarImage();
             noteBlockHolder.avatarImageView.setOnTouchListener(null);
         }
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java]: Just an FYI that I tested this and the `UserNoteBlock` change below and they're fine.

Comment (oguzkocer) [inline WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java]: Thank you!

**Comment by nbradbury** [comment]
:shipit:

**Pull request merged**